buildSrc/src/main/kotlin/kotlin-conventions.gradle.kts

```diff
@@ -1,22 +1,19 @@
-import org.gradle.api.tasks.testing.Test
-import org.gradle.api.tasks.testing.logging.TestExceptionFormat
-import org.gradle.api.tasks.testing.logging.TestLogEvent
+@file:Suppress("UnstableApiUsage")
+
 import org.gradle.kotlin.dsl.kotlin
 import org.gradle.kotlin.dsl.withType
 
 plugins {
     id("kgraphql-base")
     kotlin("jvm")
+    `java-library`
```

Comment:
Why is this needed?

Reply:
Because
> The plugin cannot be used without a JVM language plugin applied as it relies on several conventions of the `java` plugin.

https://docs.gradle.org/8.10.2/userguide/jvm_test_suite_plugin.html#sec:jvm_test_suite_usage

Reply:
No idea if `java` or `java-library` is more appropriate but we already use `java-library` so 🤷‍♂️

Reply:
But we can now remove the java-library plugin in library-conventions.